lost/src/test/java/com/mapzen/android/lost/internal/LostApiClientImplTest.java

```diff
@@ -65,7 +65,7 @@ public void disconnect_shouldRemoveGeofencingApiImpl() throws Exception {
     }
 
     @Test
-    public void disconnect_shouldUnregisterLocationUpdateListeners() throws Exception {
+    public void disconnect_shouldNotUnregisterLocationUpdateListeners() throws Exception {
```

Comment:
Are we sure this is the behavior we want? This seems like it could lead to unintentionally leaving location services running in the background.

I wonder if it would be better to remove _all_ listeners (and shutdown all engines) when disconnect is called.